test/live.js

```diff
@@ -1,12 +1,10 @@
-
 // IMPORTANT: assumes node-fetch@2
```

Comment:
update and get working as simple smoke test

Reply:
19011ca

Reply:
Need clarification on how/when to run file